src/raftstore/store/fsm/batch.rs

```diff
@@ -356,7 +356,11 @@ pub trait HandlerBuilder<N, C> {
 /// To use the system, two type of FSMs and their PollHandlers need
 /// to be defined: Normal and Control. Normal FSM handles the general
 /// task while Control FSM creates normal FSM instances.
-pub struct BatchSystem<N: Fsm, C: Fsm> {
+pub struct BatchSystem<N: Fsm, C: Fsm>
```

Comment:
I think it can be simplified to following, since where clause already bind them to `FSM`

```suggestion
pub struct BatchSystem<N, C>
```